packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps} from '@react-types/shared';
+import {ButtonContext} from './Button';
+import {ContextValue, DOMProps, Provider, SlotProps, useContextProps} from './utils';
+import {filterDOMProps} from '@react-aria/utils';
+import {Input} from './Input';
+import {LinkContext} from './Link';
+import React, {createContext, ForwardedRef, forwardRef, useRef} from 'react';
+
+export interface FileTriggerProps extends SlotProps, DOMProps, AriaLabelingProps {
+  // want to be able to pass down accept types to the input 
+  accept?: string,
+  // onChange event for the input 
+  handleChange?: React.ChangeEventHandler<HTMLInputElement>
+}
```

Comment:
yeah i think `multiple` is definitely something we'll want to include. i'm a little bit more on the fence with `capture` tho because on mobile, users will already be given the option to take a photo/video or upload a photo/video which seems to take care of `capture` in a sense

Reply:
I think it's something we should support, so we match the HTML spec. Some applications might want to be opinionated about this, and default to one or the other, so we shouldn't prevent them from that.

Reply:
okay that makes sense. i'll go ahead and add `capture` as a prop